src/pudl/metadata/resources/eia860.py

```diff
@@ -242,6 +243,7 @@
                     "_out_eia__monthly_derived_generator_attributes",
                     "out_eia__monthly_generators",
                     "core_eia860m__changelog_generators",
+                    "core_eia923__monthly_energy_storage",
```

Comment:
I have a strong desire to alphabetize all these table lists but have resisted. Is there a tool that can do this for us automatically in lots of places? Just a little nit that bothers me but is OOS.